doc/Extensions/Applications.md

```diff
@@ -168,6 +169,17 @@ extend memcached /usr/local/bin/memcached
 4. Restart snmpd on your host
 5. On the device page in Librenms, edit your host and check `Memcached` under the Applications tab.
 
+### Proxmox
+1. Copy the [proxmox script](https://github.com/librenms/librenms-agent/blob/master/agent-local/proxmox) to `/usr/local/bin` (or any other suitable location) on your remote server.
+2. Make the script executable: `chmod +x /usr/local/memcached`
+3. Edit your snmpd.conf file (usually `/etc/snmp/snmpd.conf`) and add:
+```
+extend proxmox /usr/local/bin/proxmox
+```
+(Note: if your snmpd doesn't run as root, you might have to invoke the script using sudo. `extend proxmox /usr/bin/sudo /usr/local/bin/proxmox`)
+4. Restart snmpd on your host
+5. On the device page in Librenms, edit your host and check `Proxmox` under the Applications tab.
```

Comment:
Sorry yes you are correct.

If you can fix the other couple of issues and re-base we should be good.